src/components/PageElements/PageElements.tsx

```diff
@@ -268,6 +274,10 @@ export const UpdateIcon: React.FC<{ onClick: Function }> = ({ onClick }) => (
   <Icon className="clickable" name="edit" size="large" color="blue" onClick={onClick} />
 )
 
+export const AddIcon: React.FC<{ onClick: Function }> = ({ onClick }) => (
+  <Icon className="clickable" name="add" size="large" color="blue" onClick={onClick} />
```

Comment:
Just wonder if the "plus square outline" icon might look more consistent with the existing "edit" icon?
<img width="141" alt="Screen Shot 2021-10-01 at 7 45 54 PM" src="https://user-images.githubusercontent.com/5456533/135577334-0f7d6257-2425-481b-be07-414f7588e240.png">
<img width="108" alt="Screen Shot 2021-10-01 at 7 46 10 PM" src="https://user-images.githubusercontent.com/5456533/135577338-a83ce4d3-02d9-485b-9d90-d776d1e68acd.png">

Reply:
Yes, sure. That does looks better.

Reply:
Hmmm I'll leave as is since there is a slightly difference in position with the edit button that I won't investigate now. And it's not as visible with the current item